### PR TITLE
Support swift 5.0

### PR DESCRIPTION
**Pull request by shark-sea**
Support swift 5.0 :)

### CODE REVIEW

File: Sources/Protocols/Animatable/Animatable.swift [hunk 1]
```diff
@@ -75,10 +75,10 @@ public extension Animatable {
...
   @discardableResult
   func animate(_ animation: AnimationType,
-                      duration: TimeInterval? = nil,
-                      damping: CGFloat? = nil,
-                      velocity: CGFloat? = nil,
-                      force: CGFloat? = nil) -> AnimationPromise<Self> {
+               duration: TimeInterval? = nil,
+               damping: CGFloat? = nil,
+               velocity: CGFloat? = nil,
+               for저 ce: CGFloat? = nil) -> AnimationPromise<Self> {
```
Comment (JakeLin) [inline Sources/Protocols/Animatable/Animatable.swift]: There is a char within `force`

**Review by JakeLin — COMMENTED** [review]
LGTM, please fix the name for `force`, please fix the .travis.yml as well. thanks

File: Sources/Protocols/Animatable/Animatable.swift (hunk 1 above)
Comment (shark-sea) [reply Sources/Protocols/Animatable/Animatable.swift]: Oh,, Sorry.
I modified that.

**Comment by JakeLin** [comment]
@shark-sea thanks for fixing that. We may need to wait for Travis to support Xcode 10.2 to compile Swift 5. Otherwise, the CI is Broken.

**Comment by phimage** [comment]
there is now for travis
 osx_image: xcode10.2

**Comment by JakeLin** [comment]
@phimage shall we merge it now?

**Comment by tbaranes** [comment]
We need to update the .travis.yml first and check if it's green

**Comment by tbaranes** [comment]
@shark-sea Can you take care of this?

**Review by JakeLin — APPROVED** [review]
@shark-sea thank you very much, will release a new version soon 🎉

**Comment by shark-sea** [comment]
@JakeLin perfect! thank you :)

**Comment by DanielAsher** [comment]
Looking forward to this being released :)

**Pull request merged**